servicenow/servicenow-script.txt

```diff
@@ -0,0 +1,152 @@
+(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {        
```

Comment:
May I ask why this script is a .txt file?

Reply:
Since the user just needs to copy/paste the code, I put in .txt format.  Let me know if another file format is preferred or doesn't matter.

Reply:
I think we want to keep it as the format of the source code. If it is nodejs, we may put it as .js. They should still be able to copy paste it with .js extension.

Reply:
Another thing is, maybe because of the extension, I don't think the source code is well formatted.

Reply:
ok, I'll make the edit.

Reply:
renamed file to servicenow.js